services/app-web/src/components/FilterAccordion/FilterAccordion.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react'
+import styles from './FilterAccordion.module.scss'
+import { Accordion, Button } from '@trussworks/react-uswds'
+
+export type FilterAccordionPropType = {
+    onClearFilters?: () => void
+    filterTitle: string
+    children: React.ReactElement[]
```

Comment:
🔍 suggestion: related to ensuring types are descriptive to intended use case for this component
- `onClearFilters` should be required or the button display should be conditional based on presence of this prop. This change would avoid unexpected UX on click.
- `children` seem to be `FilterSelect[]`. The component assumes children are "FilterSelect" elements that would accept  a `toggleClearFilter` prop, not a generic react element of any type.

Reply:
The intent of `onClearFilters` was surface a function that would get triggered when the `Clear filters` button is clicked. For instance if `HealthPlanPackageTable` had a function that is called when filters are cleared, you can pass that function into `FilterAccordion` to be called when the `Clear filter` button is called. The idea was to allow `FilterAccordion` component to clear its filters without having to write logic in its parent component. As for displaying the button, we can put another prop to have it show or hide.

Reply:
@JasonLin0991 Gotcha, I think I understand the intent. My comment is trying to address that display of the associated button seems dependent on that prop. We don't want unexpected UX where a button is visible but nothing happens when you click it (which could happen right now).

 To address that I would either make `onClearFilters` required  or make the button UI dependant on that prop *`onFlearFilters && <Button /> `  (probably the second one, for most flexibility). I don't think any additional props are needed...

Reply:
We paired on this in person, ✅

Reply:
Recap our zoom convo here. The `Clear filters` button is functional independently from the `onClearFilters` prop. The `onClearFilters` prop is  surfaced to the parent incase there needs to be a side effect triggered when clicking on the `Clear filters` button. 

If we wanted we could hide the button when there are no children in the component. Although the `FilterAccordion` should not be an empty accordion.